### PR TITLE
OCPBUGS-33636: Updating the secrets using Form editor displays an unknown warning message

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before:
Screenshot 2024-05-14 at 4 41 35 PM

### After:
Screenshot 2024-05-14 at 4 43 35 PM
Screenshot 2024-05-14 at 4 43 44 PM

### CODE REVIEW

**Review by parth-gr — APPROVED** [review]
lgtm

**Review by rhamilto — APPROVED** [review]
/lgtm

File: frontend/public/components/secrets/create-secret.tsx [hunk 1]
```diff
@@ -1322,7 +1322,8 @@ export const SecretLoadingWrapper = withTranslation()(
       if (!secretTypeAbstraction) {
         return <LoadingBox />;
       }
-      const fixed = _.reduce(fixedKeys, (acc, k) => ({ ...acc, k: _.get(obj.data, k) }), {});
+      const fixed = fixedKeys.reduce((acc, k) => ({ ...acc, [k]: obj.data?.[k] }), {});
```
Comment (spadgett) [inline frontend/public/components/secrets/create-secret.tsx:1325]: Is `fixedKeys` always defined?

Comment (rhamilto) [reply frontend/public/components/secrets/create-secret.tsx:1325]: Good catch, @spadgett.  @cyril-ui-developer, might need an update here.  Gonna put a hold on this.

/hold

Comment (cyril-ui-developer) [reply frontend/public/components/secrets/create-secret.tsx:1325]: @spadgett @rhamilto Not sure but looking at the [type](https://github.com/openshift/console/blob/7345d545d6060a96b9373df0527e6d074d7dc0b6/frontend/public/components/secrets/create-secret.tsx#L1376) I think so.  But I can add `?` operator it safer.

**Pull request merged**